functions/visualizations.py

```diff
@@ -113,9 +114,27 @@ def _add_antibody_metadata(
     if isinstance(assay_instances, dict):
         # only exception to list, eg olink
         assay_md = assay_instances
+    elif isinstance(assay_instances, list):
+        print(assay_instances)
+        print(file_record)
+        ds = DeepSearch(assay_instances, file_record.object_url)
+        print(ds)
+        if "matched_values" in ds:
+            if len(ds["matched_values"]) != 1:
+                raise Exception(
```

Comment:
Can you add a test that leads to this error being thrown? Same with the `TypeError` below.